dolphin/drivers/manager.py

```diff
@@ -69,7 +69,20 @@ def remove_storage(self, context, storage_id):
 
     def get_storage(self, context, storage_id):
         """Get storage device information from storage system"""
-        pass
+        # Todo : Change this when get_driver_insatnce() is ready
+        return {
+            'name': 'fake_driver',
```

Comment:
move it to fake_driver

Reply:
As you can see in the comments, Keeping it temporarily here till we have the get_driver_insatnce() implementaion ready.

Reply:
plz move it to fake driver, it's not temporraily, it's useful

Reply:
Done,